bundles/org.openhab.binding.worxlandroid/README.md

```diff
@@ -0,0 +1,649 @@
+<img src="images/landroid.svg">
+
+# WorxLandroid Binding
+
+This is the binding for Worx Landroid robotic lawn mowers. It connects openHAB with your WorxLandroid mower using the API and MQTT. This binding allows you to integrate, view and control supported Worx lawn mowers in the openHAB environment.
+
+## Supported Things
+
+Currently following Things are supported:
+
+- **Bridge Worx Landroid API** Thing representing the handler for Worx API
+- One or many Things for supported **Landroid Mower**'s
+
+## Discovery
+
+A Bridge is required to connect to the Worx API. Here you can provide your credentials for your WorxLandroid account. Once the Bridge has been added Worx Landroid mowers will be discovered automatically.
+
+## Binding Configuration
+
+Following options can be set for the **Bridge Worx Landroid API**:
+
+| Property | Description                              |
+|----------|------------------------------------------|
+| username | Username to access the WorxLandroid API. |
+| password | Password to access the WorxLandroid API. |
+
+
+Following options can be set for the **Landroid Mower**:
+
+| Property              | Description                                                                                             |
+|-----------------------|---------------------------------------------------------------------------------------------------------|
+| serialNumber          | Serial number of the mower                                                                              |
+| refreshStatusInterval | Interval for refreshing mower status (ONLINE/OFFLINE) and channel 'common#online' in seconds (min="30") |
+| pollingInterval       | Interval for polling in seconds (min="30" max="7200").                                                  |
```

Comment:
the mower.xml thing strucutre has min="0", this seems inconsistent, not sure what it should be.

Reply:
Both min are 0, no added value here. Removed.